isotovideo

```diff
@@ -53,21 +70,23 @@ use Carp 'cluck';
 use Time::HiRes qw(gettimeofday tv_interval sleep time);
 use File::Spec;
 use File::Path;
+Getopt::Long::Configure("no_ignore_case");
 
-# avoid paranoia
-$Getopt::Std::STANDARD_HELP_VERSION = 1;
+my %options;
 
-sub HELP_MESSAGE {
-    print "$0 [-d]\n";
-    print "Parses vars.json and tests the given assets/ISOS\n\n";
-    print " -d enables direct output to STDERR instead of autoinst-log.txt\n";
+sub usage {
+    eval { use Pod::Usage; pod2usage(1); };
+    if ($@) {
+        die "cannot display help, install perl(Pod::Usage)\n";
+    }
 }
 
+GetOptions(\%options, 'debug|d', 'help|h|?') or usage(1);
+usage(0) if $options{help};
```

Comment:
I have just one question, why calling `usage(1)` and `usage(0)` when the parameter doesn't seem to be used in `usage`. Was that meant to be passed to `pod2usage`?